server/src/test/java/org/elasticsearch/rest/action/info/RestClusterInfoActionTests.java

```diff
@@ -89,7 +91,12 @@ public void testHttpResponseMapper() {
                     .map(HttpStats::clientStats)
                     .map(Collection::stream)
                     .reduce(Stream.of(), Stream::concat)
-                    .toList()
+                    .toList(),
+                nodeStats.stream().map(NodeStats::getHttp).map(HttpStats::httpRouteStats).reduce(Map.of(), (l, r) -> {
```

Comment:
I think moving the two `map`s inside `reduce` is more clunky because the reduce needs to change type and Java asks you to supply both accumulator and combinator which do pretty much the same thing for this use case. I ended up moving it outside and build it separately with a for loop (see [here](https://github.com/elastic/elasticsearch/pull/99852/commits/5849e3d1e3752df71fe6e89fab0584b4f3acbcf2#diff-9b000c9b50bb2e5a9b342406b574d4d5ad5d4e2714c9f9186f9fec829e5e3de5)).